### PR TITLE
Document transcripts.

### PR DESCRIPTION
**Pull request by rosiel**
## Purpose / why
Document how transcripts work in Islandora.

## What changes were made?
A page was added outlining how transcripts can be used. 

## Verification

The claims made can be validated with an installation of islandora defaults

## Interested Parties
 @Islandora/documentation

### Pull-request Reviewer
Pull-request reviewer should ensure the following:

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

### Person Merging
The person merging should ensure the following:

* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Comment by inikolaidis** [comment]
I attempted to follow these instructions in the sandbox, not realising defaults is not a part of sandbox. An installation with Islandora Defaults is required to go through transcripts steps

**Comment by rosiel** [comment]
Thanks for testing! Yes, that is... An issue.

Transcripts should be testable in the sandbox if you add a "media track" type field in the Audio or Video media types. 

To be pedantic, Defaults is part of the Sandbox, it's just that the Install Profile clobbers over all Defaults configs with their own. Also as we are getting rid of Defaults soon, it's worth mentioning that transcripts behaviour is part of its successor, the Starter site.

File: docs/user-documentation/transcripts.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Transcripts, captions, or subtitles may be added to audio and video media.
+This functionality is available because of a field type, "Media track", 
+provided by the Islandora module, and the "Audio with Captions" and "Video 
+with Captions" field formatters from the Islandora Audio and Video modules, 
+respectively. 
+
+To add transcripts (or subtitles, or captions) to a Repository Item 
+using Islandora Defaults, navigate to the Service File media (or whichever one
+will be playing), open the edit form, and in the "Track" field, add a WebVTT file
```
Comment (inikolaidis) [inline docs/user-documentation/transcripts.md]: In order to se the "Track" field for uploading the .vtt file, I first had to alter the Structure > Media types > Edit Audio > Manage form display options to move the Track field from the disabled section to enable it. This was in the sandbox. Not sure if it makes sense to add this note before the instructions to edit the service file?

**Review by inikolaidis — APPROVED** [review]

File: docs/user-documentation/transcripts.md (hunk 1 above)
Comment (rosiel) [reply docs/user-documentation/transcripts.md]: Yep - addressed in latest commit.

**Comment by rosiel** [comment]
i'll fix the merge conflict.

**Review by kstapelfeldt — APPROVED** [review]

**Pull request merged**